force-app/main/default/classes/RD2_RecurringDonation.cls

```diff
@@ -160,11 +160,21 @@ public inherited sharing class RD2_RecurringDonation {
     }
 
     /**
-    * @description Returns True if the Next Payment Date has changed on an Elevate connected RD
-    * @return Boolean True if the Next Payment Date has changed on an Elevate connected RD
+    * @description Returns True if the Next Payment Date has changed and the value has not
+    * @return Boolean True if the Next Payment Date has changed and the value has not
     */
     public Boolean needsNextPaymentScheduleCheck() {
-        return changedFields.contains(npe03__Recurring_Donation__c.npe03__Next_Payment_Date__c);
+        return changedFields.contains(npe03__Recurring_Donation__c.npe03__Next_Payment_Date__c)
+            && !currentOrNextYearValueChanged();
```

Comment:
@andrewyu-salesforce This is what I came up with to make sure we **don't** clear the Amount when the Pause schedule changes. I don't love it, but I think it should work?